src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background: url(../images/star.svg) no-repeat center ;
+  }
+
+  @for $starCount from 1 through 5 {
+    &--#{$starCount} .stars__star:nth-child(-n + #{$starCount}) {
+      background: url(../images/star-active.svg) no-repeat center;
```

Comment:
You should specify only those properties that should change
`no-repect` and `center` properties are redundant here